src/gui/qml/FolderDelegate.qml

```diff
@@ -327,27 +327,27 @@ Pane {
                             MenuSeparator {}
 
                             MenuItem {
-                                text: folderDelegate.folder.isSyncRunning ? qsTr("Restart sync") : qsTr("Force sync now")
+                                text: folderDelegate.folder.isSyncRunning ? qsTr("Restart synchronization") : qsTr("Force synchronization now")
```

Comment:
whats wrong with "sync"? - I would always prefer the shorter version.
I thinks its okay to use the short version, seems to be common:
![screenie000342](https://github.com/user-attachments/assets/e79e71cd-b721-46c1-a7e0-a5776e72b736)
![image](https://github.com/user-attachments/assets/b60a9033-fe23-4e8d-bbf4-97b30b73feb9)

Reply:
I head the feeling it might be too technical but I guess your screenshot proves me wrong